app/src/test/java/org/simple/clinic/user/UserSessionTest.kt

```diff
@@ -98,76 +92,6 @@ class UserSessionTest {
     Analytics.clearReporters()
   }
 
-  @Test
-  fun `user approved for syncing changes should be notified correctly`() {
```

Comment:
This test was removed because we are testing this indirectly in `DataSyncTest` now.